omod/src/main/webapp/resources/scripts/jquery-ui/js/openmrsSearch.js

```diff
@@ -104,6 +104,8 @@ function OpenmrsSearch(div, showIncludeVoided, searchHandler, selectionHandler,
  *   verboseHandler: function to be called to return the text to display as verbose output
  *   attributes: Array of names for attributes types to display in the list of results
  *   showSearchButton: Boolean, indicating whether to use search button for immediate search
+ *   showVerbose: Boolean, indicates whether to check checkbox for verbose 
+ *   includeVoided: Boolean, indicates whether to check checkbox for voided 
  *   lastSearchParams: Object with properties lastSearchText, includeVoided and includeVerbose, to preserve data with browser back button
```

Comment:
You should not use request parameters, so you need to get rid of this and update the search jsp pages using them not to set them. See my comment on the ticket